src/utils/network-monitor.ts

```diff
@@ -943,7 +977,7 @@ export class NetworkMonitor {
     if (job !== undefined) {
       this.latestBlockHeight[job.network] = job.block
       if (this.verbose) {
-        this.structuredLog(job.network, `Block procesing complete ✅`, job.block)
+        this.structuredLog(job.network, `Block processing complete ✅`, job.block)
```

Comment:
nice catch on spelling.

I see that you have the update logic in the `processBlock` function instead of here.